features/check_registration.feature

```diff
@@ -17,7 +17,6 @@ Feature: Check client registration
     And I should see a "Packages:" link
     And I should see a "Edit These Properties" link
     And I should see a "[Management]" text
-    And I should see a "Add to SSM" link
```

Comment:
@MalloZup could you please revert this change from PR? I'll address the needed changes of `check_registration.feature` on my upcoming PR. That way we can merge this before even my PR.